From 8e71fa5e4d86bedfd26df85381d65d6b4c860020 Mon Sep 17 00:00:00 2001
From: YueHaibing <yuehaibing@huawei.com>
Date: Fri, 8 Feb 2019 19:24:45 +0100
Subject: fbdev: chipsfb: remove set but not used variable 'size'
Git-commit: 8e71fa5e4d86bedfd26df85381d65d6b4c860020
Patch-mainline: v5.1-rc1
References: bsc#1106886

Fixes gcc '-Wunused-but-set-variable' warning:

drivers/video/chipsfb.c: In function 'chipsfb_pci_init':
drivers/video/chipsfb.c:352:22: warning:
 variable 'size' set but not used [-Wunused-but-set-variable]

Fixes: 8c8709334cec ("[PATCH] ppc32: Remove CONFIG_PMAC_PBOOK").
Signed-off-by: YueHaibing <yuehaibing@huawei.com>
Acked-by: Michael Ellerman <mpe@ellerman.id.au>
Cc: Daniel Vetter <daniel.vetter@ffwll.ch>
Cc: Christophe Leroy <christophe.leroy@c-s.fr>
[b.zolnierkie: minor commit summary and description fixups]
Signed-off-by: Bartlomiej Zolnierkiewicz <b.zolnierkie@samsung.com>
Acked-by: Thomas Zimmermann <tzimmermann@suse.de>
---
 drivers/video/chipsfb.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/video/chipsfb.c
+++ b/drivers/video/chipsfb.c
@@ -351,7 +351,7 @@ static int __devinit
 chipsfb_pci_init(struct pci_dev *dp, const struct pci_device_id *ent)
 {
 	struct fb_info *p;
-	unsigned long addr, size;
+	unsigned long addr;
 	unsigned short cmd;
 	int rc = -ENODEV;
 
@@ -363,7 +363,6 @@ chipsfb_pci_init(struct pci_dev *dp, con
 	if ((dp->resource[0].flags & IORESOURCE_MEM) == 0)
 		goto err_disable;
 	addr = pci_resource_start(dp, 0);
-	size = pci_resource_len(dp, 0);
 	if (addr == 0)
 		goto err_disable;
 
